examples/todo/test/acceptance/todo.acceptance.ts

```diff
@@ -179,4 +193,14 @@ describe('Application', () => {
   async function givenTodoInstance(todo?: Partial<Todo>) {
     return await todoRepo.create(givenTodo(todo));
   }
+
+  // JSON transport does not preserve properties that are undefined
+  // As a result, deepEqual checks fail because the expected model
+  // value contains these undefined property values, while the actual
+  // result returned by REST API does not.
+  // Use this function to convert a model instance into a data object
+  // as returned by REST API
+  function deserializedFromJson<T extends object>(value: T): T {
```

Comment:
nitpick: not a necessary part of this PR, we can update [this test](https://github.com/strongloop/loopback-next/blob/bc9e2f6e511180e7b21975d57de0480c72c1d534/examples/todo/test/acceptance/todo.acceptance.ts#L94) to call `deserializedFromJson` as well :)